massa-consensus-exports/src/settings.rs

```diff
@@ -312,84 +323,152 @@ impl ConsensusConfig {
-                ledger_flush_interval: Some(200.into()),
-                ledger_reset_at_startup: true,
-                initial_ledger_path: initial_ledger_path.to_path_buf(),
-                operation_batch_size: 100,
-                initial_rolls_path: tempfile::tempdir()
-                    .expect("cannot create temp dir")
-                    .path()
-                    .to_path_buf(),
-                stats_timespan: 60000.into(),
-                max_send_wait: 500.into(),
-                force_keep_final_periods: 0,
-                block_db_prune_interval: 1000.into(),
-                max_item_return_count: 1000,
-                disable_block_creation: true,
-            }
-            .config()
+/// Communication async channels for the consensus worker
+/// Contains consensus channels associated (protocol & execution)
+/// Contains alse controller async channels (command, manager receivers and event sender)
+/// Contains a sender to the pool worker commands
+pub struct ConsensusWorkerChannels {
+    /// Associated protocol command sender.
+    pub protocol_command_sender: ProtocolCommandSender,
+    /// Associated protocol event listener.
+    pub protocol_event_receiver: ProtocolEventReceiver,
+    /// Associated execution event listener.
+    pub execution_event_receiver: ExecutionEventReceiver,
+    /// Execution command sender.
+    pub execution_command_sender: ExecutionCommandSender,
+    /// Associated Pool command sender.
+    pub pool_command_sender: PoolCommandSender,
+    /// Channel receiving consensus commands.
+    pub controller_command_rx: mpsc::Receiver<ConsensusCommand>,
+    /// Channel sending out consensus events.
+    pub controller_event_tx: mpsc::Sender<ConsensusEvent>,
+    /// Channel receiving consensus management commands.
+    pub controller_manager_rx: mpsc::Receiver<ConsensusManagementCommand>,
+}
+
+/// Public channels associated to the consensus module.
+/// Execution & Protocol Sender/Receiver
+pub struct ConsensusChannels {
+    pub execution_command_sender: ExecutionCommandSender,
+    pub execution_event_receiver: ExecutionEventReceiver,
+    pub protocol_command_sender: ProtocolCommandSender,
+    pub protocol_event_receiver: ProtocolEventReceiver,
+    pub pool_command_sender: PoolCommandSender,
+}
+
+const SERIALIZATION_CONTEXT: SerializationContext = SerializationContext {
+    max_block_operations: 1024,
+    parent_count: 2,
+    max_peer_list_length: 128,
+    max_message_size: 3 * 1024 * 1024,
+    max_block_size: 3 * 1024 * 1024,
+    max_bootstrap_blocks: 100,
+    max_bootstrap_cliques: 100,
+    max_bootstrap_deps: 100,
+    max_bootstrap_children: 100,
+    max_ask_blocks_per_message: 10,
+    max_operations_per_message: 1024,
+    max_endorsements_per_message: 1024,
+    max_bootstrap_message_size: 100000000,
+    max_bootstrap_pos_entries: 1000,
+    max_bootstrap_pos_cycles: 5,
+    max_block_endorsements: 8,
+};
+
+use std::path::Path;
```

Comment:
why this import is not as the top of the file ?

Reply:
I fixed that point forward with the config refactoring